src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -45,95 +39,103 @@
       - sudo
       - systemd
       - tcpdump
-      - tig
-      - tmate
-      - tmux
-      - vim
       - wget
       - which
       - wireshark-cli
+      disable_gpg_check: yes
   when: "'base_ground' in group_names"
 
 - name: Install packages for client base image
   block:
+  - name: Install ipa client
+    package:
+      state: present
+      name: '{{ ipa.client }}'
   - name: Install SSSD and its dependencies
-    dnf:
+    package:
       state: present
       name:
-      - freeipa-client
       - nfs-utils
       - realmd
       - sssd
       - sssd-*
   - name: Install debug information for selected packages
-    shell: |
-      dnf debuginfo-install -y  \
-        dbus                    \
-        glibc                   \
-        libcmocka               \
-        libdhash                \
-        libini_config           \
-        libldb                  \
-        libtalloc               \
-        libtevent
-    when: debuginfo
-  when: "'base_client' in group_names"
+    command: dnf debuginfo-install  -y  {{ item }}
+    with_items:
+      - dbus
+      - glibc
+      - libcmocka
+      - libdhash
+      - libini_config
+      - libldb
+      - libtalloc
+      - libtevent
+    ignore_errors: yes
+    when:
+     - debuginfo
+     - "'dnf' in ansible_facts.packages"
+  - name: Install packages required for passkey testing
+    package:
+      state: present
+      name:
+      - gcc
+      - openssl-devel
+      - umockdev
+    when: passkey_support
+  when: "'base_client' in group_names or 'client' in group_names or 'base_ipa' in group_names or 'ipa' in group_names"
```

Comment:
Again, avoid functional changes. This block is for client only.

Reply:
This is not present in master.

Reply:
This is present in master pulling build environment on IPA:
https://github.com/SSSD/sssd-ci-containers/blob/master/src/ansible/roles/packages/tasks/Fedora.yml#L133
I just added that it works on client and ipa without  "base_"